test-integration/helpers/ui/test-element.coffee

```diff
@@ -94,10 +97,12 @@ class TestHelper extends TestItemHelper
       defaultWaitTime: 40 * 1000 # TODO: Letting tests define their own wait time is dangerous. tutor-dev takes > 10sec to delete a task-plan
       # defaultWaitTime: 20 * 1000 # 20sec seems to be enough for deployed code but not local
 
+    @test = test
     @_options = _.assign {}, defaultOptions, options
     @_el = {}
 
     commonElements.loadingState = @options.loadingLocator
+    commonElements.self = testElementLocator
```

Comment:
I don't think every helper has a `loadingState` (& therefore a working `waitUntilLoaded` method), right?

Another advantage of having `self` & `loadingState` in the config (I think) is that the documentation might be a bit clearer than having to look for CSS selectors in 3 places (instead of 1)

Thoughts @openstax/tutor-fe ?